transaction.go

```diff
@@ -142,11 +170,16 @@ type Transactions struct {
 	Transaction []*Transaction `xml:"transaction"`
```

Comment:
I don't see this mentioned in docs but the Braintree Java SDK has a 3DS field on the [TransactionRequest](https://github.com/braintree/braintree_java/blob/master/src/main/java/com/braintreegateway/TransactionRequest.java#L305) class for the auth token.